opencog/benchmark/benchmark.py

```diff
@@ -7,12 +7,9 @@
 Displays the number of operations per second achieved under various
 scenarios utilizing the Python bindings that wrap the AtomSpace using
 Cython. Some of the tests compare the evaluation of nodes using the
-Python Scheme bindings which take much longer than their comparable
-Cython bindings. XXX Which is utterly false and insane -- the
-measurements are being done incorrectly, since the Scheme bindings
-normally run about 3x to 10x faster than the Python bindings do.
-Python is generally quite slow, and is very unlikely to be faster than
-scheme; something is borked with this test case.
+Python scheme_eval_h which takes much longer than the comparable
+Cython bindings because of the overhead of creating and initializing
+a SchemeEval object.
```

Comment:
??? Why'd you remove the comment?  When I measure python and compare it to scheme, I find that scheme is faster. So this just seems like some random partisan comment that does not reflect reality, at least, not the reality of the python 2.7.6 that I have installed on my system.